### ISSUE TITLE
How to set AR core camera preview size 

### ISSUE THREAD

**Issue by mikelhm** [google-ar/arcore-android-sdk#1260]
The camera preview size  is default 640*480,  
Frame frame = session.update();
Image cameraImage = frame.acquireCameraImage();
 int imageWidth = cameraImage.getWidth();
 int imageHeight = cameraImage.getHeight();

imageWidth = 640
imageHeight = 480

 Is there any api to  set custom preview size in ar core?? ?   just like  setting previewsize in  camera1.0  camera 2.0

**Comment by devbridie** [comment]
Please see the [Camera Configs](https://developers.google.com/ar/develop/java/camera-configs) documentation page, and filtering on [`getTextureSize()`](https://developers.google.com/ar/reference/java/com/google/ar/core/CameraConfig#getTextureSize-).

**Comment by mikelhm** [comment]
> Please see the [Camera Configs](https://developers.google.com/ar/develop/java/camera-configs) documentation page, and filtering on [`getTextureSize()`](https://developers.google.com/ar/reference/java/com/google/ar/core/CameraConfig#getTextureSize-).

![image](https://user-images.githubusercontent.com/10327642/124210431-fbcd4380-db1d-11eb-93a5-f5959811d800.png)
it seems cameraConfig.getImageSize  work.  it return some few size.   
but  not all the phone supported camera size. like camera1.0,  camera 2.0.   I mean  there are only few sizes.

is this a feature of ar core?

**Comment by devbridie** [comment]
Indeed, not all camera sizes are available in `getImageSize` due to the way ARCore currently processes images.

**Comment by mikelhm** [comment]
> Indeed, not all camera sizes are available in `getImageSize` due to the way ARCore currently processes images.

ok  i see.
